packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [3.5.0]
+* Introduce `create_distribution` command
```

Comment:
@kostadin-damyanov-prime we are about to introduce a new command as part of the rx_bloc_cli executable. The goal is to scaffold the base structure of the distribution repository. We will appreciate if you suggest a name of the command of something pops in your mind

Reply:
@DDavidPrime, please add more details to the changelog

Reply:
I though of suggesting `create_certificates_repo` but that's much longer. And I'm now having second thoughts about it, now that I read it.